SimTracker/TrackHistory/src/VertexHistory.cc

```diff
@@ -31,6 +30,14 @@ VertexHistory::VertexHistory(const edm::ParameterSet &config, edm::ConsumesColle
   quality_ = 0.;
 }
 
+void VertexHistory::fillPSetDescription(edm::ParameterSetDescription &desc) {
+  desc.addUntracked<edm::InputTag>("trackingTruth", edm::InputTag("mix", "MergedTrackTruth"));
+  desc.addUntracked<edm::InputTag>("vertexAssociator", edm::InputTag("vertexAssociatorByTracksByHits"));
+  desc.addUntracked<bool>("bestMatchByMaxValue", true);
+  desc.addUntracked<bool>("enableRecoToSim", false);
+  desc.addUntracked<bool>("enableSimToReco", true);
```

Comment:
These values are reversed with respect to the existing Python PSet: https://github.com/cms-sw/cmssw/blob/a83766bfe6fb83b7868264fd00b19f5fbfeefbb7/SimTracker/TrackHistory/python/VertexHistory_cff.py#L29-L30

Reply:
thanks, fixed.